java/ql/src/change-notes/2023-07-25-trust-boundary-violation-query.md

```diff
@@ -0,0 +1,5 @@
+---
+category: newQuery
+---
+* Added the `java/trust-boundary-violation` query to detect trust boundary violations between http requests and the http session.
```

Comment:
```suggestion
* Added the `java/trust-boundary-violation` query to detect trust boundary violations between HTTP requests and the HTTP session.
```

Also, if you end up using more generic remote sources, consider rewording this to not mention HTTP requests specifically.